forum/.htaccess

```diff
@@ -11,3 +11,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteCond %{REQUEST_FILENAME} !-d
 RewriteRule ^.*$ index.php?qa-rewrite=$0&%{QUERY_STRING} [L]
 </IfModule>
+
+<IfModule mod_headers.c>
+Header set X-Frame-Options "sameorigin"

```

Comment:
Ta reguła powinna być zduplikowana jako dyrektywa w CSP.

Reply:
Chodzi o "frame-src"?

Reply:
`frame-src` określa zasoby, jakie mogą być na stronie zamieszczane. To, gdzie strona może być zamieszczana, to `frame-ancestors`.